src/bench/bench_bitcoin.cpp

```diff
@@ -118,6 +119,9 @@ int main(int argc, char** argv)
         return EXIT_SUCCESS;
     }
 
+    std::string sha256_algo = SHA256AutoDetect();
+    tfm::format(std::cout, "Using the '%s' SHA256 implementation\n", sha256_algo);
```

Comment:
This always prints a SHA log even when no SHA related benchmark is run

Reply:
Thanks! That change has been [dropped](https://github.com/bitcoin/bitcoin/pull/27598#issuecomment-1549995815).